client/src/components/EventItem.js

```diff
@@ -442,26 +465,6 @@ const handleJoinDeadlineError = (error) => {
       setDeleteModal(false);
     });
   };
-
-  useEffect(() =>
-  {
-    const confirmLike = () => 
-    {
-      fetch(`/api/is/attending/${props.event._id}/${props.currentUser.id}` , {
-        method: 'GET'
-      })
-      .then(response => response.json())
-      .then(data => 
-        {
-          setLiking(data.attending);
-          setLoadingLikes(false);
-        }
-      );
-    }
-
-    // Find out if the user is like this event or not
-    confirmLike();
-  }, []);
```

Comment:
Also, I moved this piece of code in the above useEffect triggered when the props are changed. It results on the same effects, and the like button is no well updated